include/yaml-cpp/traits.h

```diff
@@ -100,4 +103,17 @@ template <class Cond, class T = void>
 struct disable_if : public disable_if_c<Cond::value, T> {};
 }
 
+template <typename S, typename T>
+class is_streamable {
+  template <typename SS, typename TT>
```

Comment:
Flip these around, so the public methods are before the private ones.

Reply:
I changed it to struct and made everything public, because `is_streamable.test` must be declared before `is_streamable.value`.